pyDeltaRCM/deltaRCM_driver.py

```diff
@@ -3,122 +3,32 @@
-                                            'type': 'long', 'default': 50},
-        'coeff__surface_smoothing': {'name': 'Csmooth',
-                                     'type': 'float', 'default': 0.9},
-        'coeff__under_relaxation__water_surface': {'name': 'omega_sfc',
-                                                   'type': 'float', 'default': 0.1},
-        'coeff__under_relaxation__water_flow': {'name': 'omega_flow',
-                                                'type': 'float', 'default': 0.9},
-        'coeff__iterations_smoothing_algorithm': {'name': 'Nsmooth',
-                                                  'type': 'long', 'default': 5},
-        'coeff__depth_dependence__water': {'name': 'theta_water',
-                                           'type': 'float', 'default': 1.0},
-        'coeff__depth_dependence__sand': {'name': 'coeff_theta_sand',
-                                          'type': 'float', 'default': 2.0},
-        'coeff__depth_dependence__mud': {'name': 'coeff_theta_mud',
-                                         'type': 'float', 'default': 1.0},
-        'coeff__non_linear_exp_sed_flux_flow_velocity': {'name': 'beta',
-                                                         'type': 'long', 'default': 3},
-        'coeff__sedimentation_lag': {'name': 'sed_lag',
-                                     'type': 'float', 'default': 1.0},
-        'coeff__velocity_deposition_mud': {'name': 'coeff_U_dep_mud',
-                                           'type': 'float', 'default': 0.3},
-        'coeff__velocity_erosion_mud': {'name': 'coeff_U_ero_mud',
-                                        'type': 'float', 'default': 1.5},
-        'coeff__velocity_erosion_sand': {'name': 'coeff_U_ero_sand',
-                                         'type': 'float', 'default': 1.05},
-        'coeff__topographic_diffusion': {'name': 'alpha',
-                                         'type': 'float', 'default': 0.1},
-        'basin__opt_subsidence': {'name': 'toggle_subsidence',
-                                  'type': 'choice', 'default': False},
-        'basin__maximum_subsidence_rate': {'name': 'sigma_max',
-                                           'type': 'float', 'default': 0.000825},
-        'basin__subsidence_start_timestep': {'name': 'start_subsidence',
-                                             'type': 'float', 'default': 0},
-        'basin__opt_stratigraphy': {'name': 'save_strata',
-                                    'type': 'choice', 'default': False}
-    }
-
-    def __init__(self, input_file):
+    #############################################
+    ################## __init__ #################
+    #############################################
+
+    def __init__(self, **kwargs):
         """
         Creates an instance of the model
 
         Sets the most commonly changed variables here
         Calls functions to set the rest and create the domain (for cleanliness)
         """
-
+        
```

Comment:
?

Reply:
That must be a stranded tab. Probably happened when I was manually resolving the merge.